zipkin-storage/elasticsearch-http/src/main/java/zipkin/storage/elasticsearch/http/HttpClient.java

```diff
@@ -289,23 +322,18 @@ public void clear(String index) {
     checkState(!"RED".equalsIgnoreCase(status), "Health status is RED");
   }
 
-  private static final class IndicesHealth extends GenericResultAbstractAction {
+  private static final class IndicesHealth extends Health {
```

Comment:
I had this change locally because, as I'm sure you noted, my previous attempt to move the method-local class out didn't quite work (the constructor needed the additional setURI call).

The worst part about this is that I think it means the ES tests didn't run for my last PR – all the tests are passing consistently for me locally, but maybe having some trouble in CI. I might need your help to dig into that if it persists.